expressions.js

```diff
@@ -0,0 +1,169 @@
+
+//
+// expressions
+//
+
+const expressionsMap = {
```

Comment:
This is map of expressions and functions that are executed to replace them with real data